tests/odm/documents/test_bulk_write.py

```diff
@@ -182,3 +182,31 @@ async def test_native_upsert_not_found(documents, document_not_inserted):
         await bulk_writer.commit()
 
     assert await DocumentTestModel.count() == 6
+
+
+async def test_different_models_same_collection(document):
+    try:
+        async with BulkWriter() as bulk_writer:
+            await One.insert_one(One(), bulk_writer=bulk_writer)
+            await Two.insert_one(Two(), bulk_writer=bulk_writer)
+    except ValueError:
+        pytest.fail("ValueError raised unexpectedly")
+
+
+async def test_empty_operations(documents_not_inserted):
+    bulk = BulkWriter()
+    assert await bulk.commit() == None
+
+
+async def test_comment(documents_not_inserted):
```

Comment:
```suggestion
async def test_comment():
```
Nothing is being inserted to the DB, so the `documents_not_inserted` fixture is not needed.
Test is passing even without it.